ra2ce/analyses/direct/analyses_direct.py

```diff
@@ -141,17 +152,17 @@ def road_damage(self, analysis: dict) -> gpd.GeoDataFrame:
 
             return event_gdf.gdf
 
-        elif analysis["event_type"] == "return_period":
+        elif analysis.event_type == "return_period":
```

Comment:
is this later on marked as  "convert to enum" ?

Reply:
Done. There are many candidates. Added to the backlog.